app/src/components/TokenCard.tsx

```diff
@@ -0,0 +1,23 @@
+
+import React from 'react';
+
+import { Box, Direction, Image, PaddingSize, Spacing, Stack, Text, TextAlignment } from '@kibalabs/ui-react';
+
+export const TokenCard = (): React.ReactElement => {
+  return (
+    <Stack direction={Direction.Vertical} isFullHeight={true} isFullWidth={true} padding={PaddingSize.Wide}>
+      <Text variant='header3-singleLine'>Your Holdings</Text>
+      <Spacing variant={PaddingSize.Wide} />
+      <Box variant='card' height='204px' width='140px'>
+        <Box height='140px' width='140px'>
+          <Image source={'/assets/icon.png'} alternativeText='image' fitType='contain' />
+        </Box>
+        <Spacing variant={PaddingSize.Wide} />
+        <Text variant='subtitle' alignment={TextAlignment.Center}>mdtp</Text>
+        <Text variant='small' alignment={TextAlignment.Center}>bought on 12/12/21 for 0.3</Text>
```

Comment:
there is so much hardcoded in the sentence that is going to take you ages to figure out - its better to not have this and add a subtask and then do it than do this - when i see the picture i think you're done but when i look at the code its almost like everything is still to be done